### ISSUE TITLE
Can't upload zip file into the BloodHound Web GUI

### ISSUE THREAD

**Issue by dn9uy3n** [SpecterOps/BloodHound#10]
I installed BloodHound CE on Kali Linux. According to the docs, Drag and drop zip file into the BloodHound GUI, and the interface will merge the data into the database. However, it doesn't work and I can't find anywhere else on the web to upload files
![image](https://github.com/SpecterOps/BloodHound/assets/21336403/ad06ca70-4fa3-47bb-a6bc-9270065b7613)

**Comment by zinic** [comment]
We've got some ideas around how to implement this. There's a few +1 for this issue from the community Slack channel as well.

We're due some internal discussion for priority before I can give any kind of ETA or acceptance of the feature request but I will respond here with our decision later today.

**Comment by noraj** [comment]
In terms of ergonomic, coming from BH v4 I searched for the upload feature for ages.

I searched the upload data button on the UI for 10 minutes feeling stupid. Then search in all the sections of the official documentation https://support.bloodhoundenterprise.io/hc/en-us where there is documentation for data collection but not for data upload / ingestion. It's only taking a look at all issues there with the keyword `upload` (and not import) that I found that it's either drag and drop a zip or select and upload json files at http://127.0.0.1:8080/ui/administration/file-ingest. 

![image](https://github.com/SpecterOps/BloodHound/assets/16578570/596e0007-c6d1-4d1f-9965-91720efc009b)

It would be more straightforward and satisfy muscle memory to get back the quick access feature bar like in BH v4

https://cdn-0.securityonline.info/ezoimgfmt/camo.githubusercontent.com/ed8436812674fefc493bcc9336b087d6395d7757/687474703a2f2f692e696d6775722e636f6d2f6c74623570316b2e706e67?ezimgfmt=rs:202x484/rscb1/ng:webp/ngcb1
